pkg/universe.dagger.io/aws/aws.cue

```diff
@@ -90,15 +107,32 @@ import (
 			if credentials.sessionToken != _|_ {
 				AWS_SESSION_TOKEN: credentials.sessionToken
 			}
+
+			if credentials.profile != _|_ {
+				AWS_PROFILE: credentials.profile
+			}
+
+			if credentials.containerCredentialsRelativeUri != _|_ {
+				AWS_CONTAINER_CREDENTIALS_RELATIVE_URI: credentials.containerCredentialsRelativeUri
+			}
 		}
 
 		if configFile != _|_ {
-			mounts: aws: {
+			mounts: awsConfigFile: {
 				contents: configFile
 				dest:     "/aws"
 				ro:       true
 			}
 			env: AWS_CONFIG_FILE: "/aws/config"
 		}
+
+		if configFolder != _|_ {
+			mounts: awsConfigFolder: {
+				contents: configFolder
+				dest:     "/aws"
+				ro:       true
+			}
+			env: AWS_CONFIG_FILE: "/aws/config"
+		}
```

Comment:
Is there a case where both configurations are filled? If not, you should enforce the if statement to allow only mounting one kind of configuration so it will avoid unexpected behaviour in case of wrong usage (providing both config file and folder)

Reply:
No, there isn't. `configFolder` should have precedence over configFile.

Reply:
Did you test it? I do not know if that will be true in action